MOBILE_API.xml

```diff
@@ -2447,12 +2496,15 @@
     </struct>
 
     <struct name="ButtonCapabilities" since="1.0">
-        <description>Contains information about a button's capabilities.</description>
+        <description>
+            Contains information about a button's capabilities.
+            NOTE: Multiple button capabilities can exist for the same ButtonName with different moduleIds.
```

Comment:
```suggestion
            Contains information about a button's capabilities.
            NOTE: Multiple button capabilities can exist for the same ButtonName with different module ids. The module Id for a button should match the module id for a "primary" module type, such as a climate module or a radio module. Whichever module the id matches the button's id is the module the button will control.
```

Added additional info regarding something the app lib teams were confused about a few months ago.